extensions/common/vault/vault-aws/src/main/java/org/eclipse/edc/vault/aws/AwsSecretsManagerVaultExtension.java

```diff
@@ -33,8 +36,14 @@
 public class AwsSecretsManagerVaultExtension implements ServiceExtension {
     public static final String NAME = "AWS Secrets Manager Vault";
 
-    @Setting
-    private static final String VAULT_AWS_REGION = "edc.vault.aws.region";
+    @Setting(key = "edc.vault.aws.region",
+            description = "The AWS Secrets Manager client will point to the specified region")
+    String vaultRegion;
```

Comment:
`AwsClientBuilder#region` can identify the endpoint automatically, so it might be better to support `required = false`
https://github.com/aws/aws-sdk-java-v2/blob/891cf4fb6f37ac671fa91663c23965585fbdd504/core/aws-core/src/main/java/software/amazon/awssdk/awscore/client/builder/AwsClientBuilder.java#L85-L97

Reply:
yes, but to be tackled eventually in a separated issue/PR.